enhancements/sig-architecture/6-placements/README.md

```diff
@@ -0,0 +1,470 @@
+    kind: Placement
+    metadata:
+      name: placement1
+      namespace: ns1
+    spec:
+      predicates:
+        - labelSelector:
+            matchLabels:
+              vendor: OpenShift
+    ```
+
+4. PlacementController generates placementdecisions in the same namespace according to the ManagedClusterSetBindings in this namespace and the spec of the placement.
+    ```yaml
+    apiVersion: cluster.open-cluster-management.io/v1alpha1
+    kind: PlacementDecision
+    metadata:
+      labels:
+        placement.open-cluster-management.io: placement1
+      name: placement1-decision-1
+      namespace: ns1
+    spec:
+      decisions:
+        - clusterName: cluster1
+          available: true
+        - clusterName: cluster2
+          available: false
+        - clusterName: cluster3
+          available: true
+    ```
+
+Limitations:
+- A ManagedClusterSetBinding binds only one ManagedClusterSet to a namespace. It becomes worse if a user, like admin, wants to bind all ManagedClusterSets to a namespace, because,
+  - The number of ManagedClusterSets might be very large;
+  - The new added ManagedClusterSets cannot be bound to the namespace automatically;
+
+### Workflows
+
+#### Workflow 1: Placement Reconciliation
+1. If the placement has old placementdicisions, fetch them all and get a slice of old matched clusters;
+2. Evaluate each cluster predicates (see Workflow 2 below for more details) and get the union of all predicates result;
+3. Enforce the affinity constrains on the union and remove clusters if necessary;
+4. If the placement has no old placementdicisions yet, create placementdicisions;
+5. Otherwise, update the existing placementdicisions, and add/remove placementdicisions if necessary.
+
+#### Workflow 2: Evaluate a cluster predicate
+1. If `ClusterName` is specified, return cluster if it is exists;
+2. If `ClusterSets` is not empty, get clusters belongs to intersection of `ClusterSets` and the ManagedClusterSets bound to the placement namespace;
+3. Otherwise, get clusters from the ManagedClusterSets bound to the placement namespace;
+4. If `RequiredDuringSchedulingRequiredDuringExecution` is specified, filter clusters with label/claim selectors; 
+5. Enforce the affinity constrains on clusters if exist;
+6. If `NumberOfClusters` is not specified, return all matched clusters;
+7. Otherwise, get the intersection of old matched clusters and new matched clusters;
+8. If `NumberOfClusters` is equal to the size of the intersection, return clusters in the intersection;
+9. if `NumberOfClusters` is less than the size of the intersection, select random clusters from the intersection and return;
+10. Otherwise, return clusters in the intersection with some random additional clusters from the remaining of the new matched clusters;
+
+**Note:** The old matched clusters are preferred during the predicate evaluation, which makes the placement decisions as stable as possible.
+
+### Graduation Criteria
```

Comment:
This would need explicitly indicate criteria of:

Alpha -> Beta
Beta  -> GA

Reply:
Updated as suggested.